packages/components/src/popover/index.tsx

```diff
@@ -390,100 +392,67 @@ const UnconnectedPopover = (
-	// parts of the `ColorPicker` UI (this is possible because the `ColorPicker`
-	// component exposes the `onPickerDragStart` and `onPickerDragEnd` props
-	// via internal context).
-	// While the user is performing a pointer drag, the `Popover` will render
-	// a transparent backdrop element that will serve as a "pointer events trap",
-	// making sure that no pointer events reach any potential `iframe` element
-	// underneath (like, for example, the editor canvas in the WordPress editor).
-	const [ showBackdrop, setShowBackdrop ] = useState( false );
-	const contextValue = useMemo(
-		() => ( {
-			ColorPicker: {
-				onPickerDragStart() {
-					setShowBackdrop( true );
-				},
-				onPickerDragEnd() {
-					setShowBackdrop( false );
-				},
-			},
-		} ),
-		[]
-	);
-
 	let content = (
-		<>
-			{ showBackdrop && (
-				<div
-					className="components-popover-pointer-events-trap"
-					aria-hidden="true"
-					onClick={ () => setShowBackdrop( false ) }
-				/>
+		<motion.div
+			className={ classnames( className, {
+				'is-expanded': isExpanded,
+				'is-positioned': isPositioned,
+				// Use the 'alternate' classname for 'toolbar' variant for back compat.
+				[ `is-${
+					computedVariant === 'toolbar'
+						? 'alternate'
+						: computedVariant
+				}` ]: computedVariant,
+			} ) }
+			{ ...animationProps }
+			{ ...contentProps }
+			ref={ mergedFloatingRef }
+			{ ...dialogProps }
+			tabIndex={ -1 }
+		>
+			{ /* Prevents scroll on the document */ }
+			{ isExpanded && <ScrollLock /> }
+			{ isExpanded && (
+				<div className="components-popover__header">
+					<span className="components-popover__header-title">
+						{ headerTitle }
+					</span>
+					<Button
+						className="components-popover__close"
+						icon={ close }
+						onClick={ onClose }
+					/>
+				</div>
 			) }
-			<motion.div
-				className={ classnames( 'components-popover', className, {
-					'is-expanded': isExpanded,
-					'is-positioned': isPositioned,
-					// Use the 'alternate' classname for 'toolbar' variant for back compat.
-					[ `is-${
-						computedVariant === 'toolbar'
-							? 'alternate'
-							: computedVariant
-					}` ]: computedVariant,
-				} ) }
-				{ ...animationProps }
-				{ ...contentProps }
-				ref={ mergedFloatingRef }
-				{ ...dialogProps }
-				tabIndex={ -1 }
-			>
-				{ /* Prevents scroll on the document */ }
-				{ isExpanded && <ScrollLock /> }
-				{ isExpanded && (
-					<div className="components-popover__header">
-						<span className="components-popover__header-title">
-							{ headerTitle }
-						</span>
-						<Button
-							className="components-popover__close"
-							icon={ close }
-							onClick={ onClose }
-						/>
-					</div>
-				) }
-				<div className="components-popover__content">
-					<ContextSystemProvider value={ contextValue }>
-						{ children }
-					</ContextSystemProvider>
+			<div className="components-popover__content">
+				<ContextSystemProvider value={ defaultContextValue }>
```

Comment:
Is this provider still necessary? Can we just remove it?

Reply:
Done in 19c81f73934b0a970d4de350c2ca773f0381309e and everything seems fine. Glad you spotted it.